content/tutorials/publish-subscribe.textile

```diff
@@ -113,6 +159,18 @@ blang[ruby].
 
   "See this step in Github":https://github.com/ably/tutorials/commit/75becbf
 
+blang[php].
+  Update @index.php@ and add code below before @</script>@ tag:
+
+  ```[javascript]
+    var channel = realtime.channels.get("sport");
+    channel.subscribe(function(msg) {
+      alert("Received: " + JSON.stringify(msg.data));
```

Comment:
This appears to be broken because you stringify the message data yet you are publishing a string.  This is almost certainly wrong.  Either publish JSON or don't stringify here.